### DISCUSSION TITLE
Empty Assignment & Knowledge check sections?

### DISCUSSION THREAD

**Discussion by Asartea** [TheOdinProject/curriculum#25168] [General]
Over the past few days I've seen a few people get confused that some lessons (such as https://www.theodinproject.com/lessons/foundations-installing-node-js) have the Assignment & Knowledge check headers, but don't have any actual content in those sections, because it doesn't make sense. Based on my reading of the [Layout guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lesson-layout) this is in compliance with the current layout guide, which seems to suggest those should always be there. I assume that this was done as a way to prompt people to add to these lessons as a signpost of hey there is more that could be done here.

For some lessons however, such as this one, that doesn't make any sense however, as there is no assignment or knowledge check needed for this lesson. What is the official guidance on those headers for these types of lessons?

**Comment by KevinMulhern** [comment]
Thanks for opening this @Asartea.

Makes a lot of sense to me. I'd support removing those sections. But would like to get @thatblindgeye's take on this first. Just incase theres a good reason for keeping them that we're not aware of.

**Comment by thatblindgeye** [comment]
Yep, those sections should be removed if there is no actual content. The Layout Style Guide should mention what sections should/can be deleted and when, vs what sections should always be present (most of the time)
